src/spark_manager.py

```diff
@@ -267,8 +270,10 @@ def _create_master_deployment(
                                 ],
                                 "env": env,
                                 "resources": {
-                                    "requests": {"memory": memory, "cpu": str(cores)}
+                                    "requests": {"memory": memory, "cpu": str(cores)},
                                     # TODO: Add resource limits if needed
```

Comment:
is this TODO still relevant?

Reply:
yea. yea. It's still relevant. Current `limit` is exactly the same as `requests`. I am planning to make this configurable.

Reply:
:+1: